master/reference/policy-controller/configuration.md

```diff
@@ -13,57 +13,35 @@ section.
 The policy controller supports the following environment variables to configure
 etcd access:
 
-* `ETCD_ENDPOINTS`: The list of etcd nodes in your cluster. e.g `http://10.0.0.1:2379,http://10.0.0.2:2379`
-* `ETCD_CA_CERT_FILE`: The full path to the CA certificate file for the Certificate Authority that signed the etcd server key/certificate pair.
-* `ETCD_CERT_FILE`: The full path to the client certificate file for accessing the etcd cluster.
-* `ETCD_KEY_FILE`: The full path to the client key file for accessing the etcd cluster.
+| Environment   | Description | Schema |
+| ------------- | ----------- | ------ |
+| ETCD_ENDPOINTS    | The list of etcd nodes in your cluster. e.g `http://10.0.0.1:2379,http://10.0.0.2:2379`
+| ETCD_CA_CERT_FILE | The full path to the CA certificate file for the Certificate Authority that signed the etcd server key/certificate pair. | path
+| ETCD_CERT_FILE    | The full path to the client certificate file for accessing the etcd cluster. | path
+| ETCD_KEY_FILE     | The full path to the client key file for accessing the etcd cluster. | path
 
-> **Note**: When running etcd with TLS enabled, the addresses in 
-> `ETCD_ENDPOINTS` must be hostname values such as `etcd-host:2379`, 
-> NOT IP addresses.
-{: .alert .alert-info}
+The `*_FILE` variables are _paths_ to the corresponding certificates / keys. As such, when the policy controller is running as a Kubernetes pod, you
+must ensure that the files exist within the pod. This is usually done in one of two ways:
 
-The `*_FILE` variables are _paths_ to the corresponding certificates / keys.  As such, when the policy controller is running as a Kubernetes pod, you
-must ensure that the files exist within the pod.  This is usually done in one of two ways:
-
-* Mount the certificates from the host.  This requires that the certs be present on the host that the policy controller is scheduled to / running on.
+* Mount the certificates from the host. This requires that the certs be present on the host that the policy controller is scheduled to / running on.
```

Comment:
I would suggest spelling out certs in the second sentence (as it is in the first sentence), so it would be certs --> certificates in sentence 2.

Reply:
For me, it is a little confusing to equate "scheduled to" with "running on". I guess that is primarily because "scheduled to" indicates something that will happen in the future and "running on" indicates the opposite. Also, I often see these more collapsed as "scheduled to run on." So, I personally cannot quite parse this construction but perhaps it's because I am relatively new to Kubernetes.

Reply:
Yep, I agree.  I've re-worded as "This requires that the certs be present on the host running the policy controller"